rules/integrations/azure/defense_evasion_suppression_rule_created.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2021/08/27"
+maturity = "production"
+updated_date = "2021/12/14"
+integration = "azure"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when a suppression rule is created in Azure. Adversaries can use suppression rules to prevent system administators 
+from getting legit alerts, suppression rules can help adversaries to evade detection.
+"""
+false_positives = [
+    """
+    Suppression Rule being created may be performed by a system administrator. Verify whether the user identity, 
+    user agent, and/or hostname should be making changes in your environment. Suppression Rule created from unfamiliar
+    users should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+from = "now-25m"
+index = ["filebeat-*", "logs-azure*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Azure Suppression Rule Created"
+note = """## Config
+
+The Azure Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations",
+    "https://docs.microsoft.com/en-us/rest/api/securitycenter/alerts-suppression-rules/update",
+]
+risk_score = 21
+rule_id = "f0bc081a-2346-4744-a6a4-81514817e888"
+severity = "low"
+tags = ["Elastic", "Cloud", "Azure", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:azure.activitylogs and azure.activitylogs.operation_name:"MICROSOFT.SECURITY/ALERTSSUPPRESSIONRULES/WRITE" and 
+event.outcome: "success"
```

Comment:
Tested this one, seems that modifications are included in this, not just creations